scipy/linalg/special_matrices.py

```diff
@@ -1036,3 +1037,154 @@ def dft(n, scale=None):
     elif scale == 'n':
         m /= n
     return m
+
+
+def fiedler(a):
+    """Returns a symmetric Fiedler matrix ``F[i, j] = np.abs(a[i] - a[j])``
+
+    Fiedler matrices have zero diagonals and nonnegative entries. A Fiedler
+    matrix has a dominant positive eigenvalue and other eigenvalues are
+    negative. Although not valid generally, for certain inputs, the inverse
+    and the determinant can be derived explicitly as given in [1]_.
+
+    Parameters
+    ----------
+    a : (n,) array_like
+        coefficient array
+
+    Returns
+    -------
+    F : (n, n) ndarray
+
+    See Also
+    --------
+    circulant, toeplitz
+
+
+    .. versionadded:: 1.13.0
```

Comment:
This belongs under `Notes`, that will be the issue you were seeing with the doc build

Reply:
and 1.3.0 not 1.13.0